RoddyCore/src/de/dkfz/roddy/config/ConfigurationFactory.groovy

```diff
@@ -192,6 +194,27 @@ public class ConfigurationFactory {
         return availableConfigurationsByType.get(type, []);
     }
 
+    public static String loadAndPreprocessTextFromFile(File file) {
+        if (file.name.endsWith(".xml")) // Default behaviour
+            return file.text
+
+        if (file.name.endsWith(".sh")) // Easy Bash importer
+            return loadAndPreprocessBashFile(file.text)
+
+        if (file.name.endsWith(".yml")) // YAML import
+            return loadAndPreprocessYAMLFile(file.text)
+
+        throw new IOException("Unknown file type ${file.name}")
```

Comment:
I think it is a good idea to start creating our own exceptions. I'll create one for this case.